tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
With this PR or your suggested changes, how are PrimitiveParserByteSpanToUInt32_VariableLength and PrimitiveParserByteSpanToUInt32 different other than different input strings? Isn't the first supposed to be doing.